contracts/staking/contracts/src/stake/MixinStakeBalances.sol

```diff
@@ -53,6 +53,17 @@ contract MixinStakeBalances is
         return _balanceOfOwnerInZrxVault(owner);
     }
 
+    /// @dev Returns the total stake in a given status for the entire staking system.
+    /// @param status of stake being queried.
+    /// @return Total amount of stake in given status.
+    function getGlobalStakeInStatus(IStructs.StakeStatus status)
+        public
+        view
+        returns (uint256)
+    {
+        return globalStakeByStatus[uint8(status)];
```

Comment:
I feel like we can just remove this function and make `globalStakeByStatus` public (I'm going to be making that change throughout the staking contracts).